lib/results.js

```diff
@@ -22,6 +22,7 @@ function Results () {
     this.pass = 0;
     this._stream = through();
     this.tests = [];
+    this._only = 0;
```

Comment:
Okay, just checked through them. Maybe I was confusing those with another project. I'll fix. To minimize the chance of me screwing up, it would help for you to point me to the right way to update a PR. I screwed up a PR on a project once so that thereafter every single PR indicated failed merges despite merges really being clean. I'll spend an hour researching before making an attempt. I've gotten git-shy.